lib/server/worker.js

```diff
@@ -52,6 +52,16 @@ exports.create = function() {
     }
   });
 
+  server.route({
+    method: 'DELETE',
+    path: '/a/{id}',
+    config: {
+      handler: function delete(req, reply) {
+        img.delete(req.params.id).done(reply, reply);
```

Comment:
Not sure if I'm reading this right... but does this mean I can delete _**any**_ avatar from S3 if I know it's ID? Or is there logic in there to limit it to only being able to delete your own profile image for the currently logged in user?